.github/workflows/validate-generated-files.yml

```diff
@@ -0,0 +1,23 @@
+name: validate-generated-files
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+  validate-generated-files:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: 'go.mod'
+
+      - name: Check generated files
+        run: |
+          export PATH=$PATH:$(go env GOPATH)/bin
+          make install-tools generate
+          git diff --exit-code --numstat
```

Comment:
Nitpick: might be good to print the diff as well.

Reply:
@hariso i see you shipped this one on https://github.com/ConduitIO/conduit-connector-postgres/pull/131/files#diff-721a78cbc4708b9650ff1a84a8dcbc1a5c166b5a9c2e0b1f648a952c9e21a35d, so I'll rebase.